tests/testthat/test-poisson-reg.R

```diff
@@ -1,222 +1,13 @@
-
-  glmnet_nlam <- poisson_reg() %>% set_engine("glmnet", nlambda = 10)
-  expect_equal(
-    translate(glmnet_nlam)$method$fit$args,
-    list(
-      x = expr(missing_arg()),
-      y = expr(missing_arg()),
-      weights = expr(missing_arg()),
-      nlambda = new_empty_quosure(10),
-      family = "poisson"
-    )
-  )
-
-  stan_samp <- poisson_reg() %>% set_engine("stan", chains = 1, iter = 5)
-  expect_equal(
-    translate(stan_samp)$method$fit$args,
-    list(
-      formula = expr(missing_arg()),
-      data = expr(missing_arg()),
-      weights = expr(missing_arg()),
-      chains = new_empty_quosure(1),
-      iter = new_empty_quosure(5),
-      family = expr(stats::poisson)
-    )
-  )
-})
-
-
-test_that("updating", {
-  expr1     <- poisson_reg() %>% set_engine("glm", model = FALSE)
-  expr1_exp <- poisson_reg(mixture = 0) %>% set_engine("glm", model = FALSE)
-
-  expr2     <- poisson_reg(mixture = varying()) %>% set_engine("glmnet")
-  expr2_exp <- poisson_reg(mixture = varying()) %>% set_engine("glmnet", nlambda = 10)
-
-  expr3     <- poisson_reg(mixture = 0, penalty = varying()) %>% set_engine("glmnet")
-  expr3_exp <- poisson_reg(mixture = 1) %>% set_engine("glmnet")
-
-  expr4     <- poisson_reg(mixture = 0) %>% set_engine("glmnet", nlambda = 10)
-  expr4_exp <- poisson_reg(mixture = 0) %>% set_engine("glmnet", nlambda = 10, pmax = 2)
-
-  expr5     <- poisson_reg(mixture = 1) %>% set_engine("glmnet", nlambda = 10)
-  expr5_exp <- poisson_reg(mixture = 1) %>% set_engine("glmnet", nlambda = 10, pmax = 2)
-
-  expect_equal(update(expr1, mixture = 0), expr1_exp)
-  expect_equal(update(expr3, mixture = 1, fresh = TRUE), expr3_exp)
-
-  param_tibb <- tibble::tibble(mixture = 1 / 3, penalty = 1)
-  param_list <- as.list(param_tibb)
-
-  expr4_updated <- update(expr4, param_tibb)
-  expect_equal(expr4_updated$args$mixture, 1 / 3)
-  expect_equal(expr4_updated$args$penalty, 1)
-  expect_equal(expr4_updated$eng_args$nlambda, rlang::quo(10))
-
-  expr4_updated_lst <- update(expr4, param_list)
-  expect_equal(expr4_updated_lst$args$mixture, 1 / 3)
-  expect_equal(expr4_updated_lst$args$penalty, 1)
-  expect_equal(expr4_updated_lst$eng_args$nlambda, rlang::quo(10))
 })
 
-test_that("bad input", {
-  expect_error(poisson_reg(mode = "classification"))
-  expect_error(translate(poisson_reg(), engine = "wat?"))
-  expect_error(translate(poisson_reg(), engine = NULL))
+test_that('bad input', {
+  expect_error(poisson_reg(mode = "bogus"))
+  expect_error(translate(poisson_reg(mode = "regression"), engine = NULL))
   expect_error(translate(poisson_reg(formula = y ~ x)))
-  expect_error(translate(poisson_reg(x = seniors[, 1:3], y = factor(seniors$count)) %>% set_engine("glmnet")))
-  expect_error(translate(poisson_reg(formula = y ~ x) %>% set_engine("glm")))
-})
-
-test_that("printing", {
-  expect_output(print(poisson_reg()))
-})
-
-
-# ------------------------------------------------------------------------------
-
-test_that("glm execution", {
```

Comment:
I'm not sure if the tests below here are covered in parsnip tests—this may be poissonreg functionality worth checking.